web/index.html

```diff
@@ -444,15 +446,17 @@ <h2>Adopting MCTS with LLMs</h2>
     </p>
     <p>
       In the <b>selection</b> phase, the MCTS algorithm makes a decision on which node or branch it
-      should explore next by assigning a kind of "priority" to each node. Greater weight is
-      typically given to nodes that are either higher in value or have fewer visits. For our case of
+      should explore or exploit next by assigning a kind of "priority" to each node. Greater weight is
+      typically given to nodes that are either higher in value (currently observed performance) or have fewer visits (not yet explored i.e. uncertainity). For our case of
       LLM code generation, the exact calculation of this priority is given by:
     </p>
     <img loading="lazy" src="public/p-ucb.png" alt="p-ucb function" class="p-ucb" />
     <p>and the node with the highest priority is selected:</p>
     <img loading="lazy" src="public/p-ucb-select.png" alt="p-ucb selection" class="p-ucb-select" />
     <p>
-      It is not necessary to understand the above in detail. What's more important is to realise
+      This formulation adapts a popular Upper Confidence Bound metric in RL with transfomer probabilities to weight the exploration, see more in <a href="https://arxiv.org/abs/2303.05510" target="_blank"
```

Comment:
Suggestion:

`
This formulation adapts the upper confidence bound heuristic with transformer probabilities to weight the exploration, as described in Section D.1 of <a href="https://arxiv.org/abs/2303.05510" target="_blank">Planning with Large Language Models for Code Generation</a>. The choice of which nodes to explore is thus a function of ....
`